exec/plugin-commands-script-runners/test/dlx.e2e.ts

```diff
@@ -115,21 +115,28 @@ test('dlx should not fail when the installed package has many commands and one e
   expect(fs.existsSync('touch.txt')).toBeTruthy()
 })
 
-test('dlx --package <pkg1> [--package <pkg2>]', async () => {
-  prepareEmpty()
+// Creating a describe block to scope the jest.retryTimes setting.
+describe('dlx on flaky test', () => {
+  // This test is more prone to network flakes since it reaches out to
+  // github.com for a dependency.
+  jest.retryTimes(1, { logErrorsBeforeRetry: true })
 
-  await dlx.handler({
-    ...DEFAULT_OPTS,
-    dir: path.resolve('project'),
-    storeDir: path.resolve('store'),
-    cacheDir: path.resolve('cache'),
-    package: [
-      'zkochan/for-testing-pnpm-dlx',
-      'is-positive',
-    ],
-  }, ['foo'])
+  test('dlx --package <pkg1> [--package <pkg2>]', async () => {
```

Comment:
From the description of the test it doesn't look like the goal is testing git-hosted dependencies. So, I guess we can just publish `zkochan/for-testing-pnpm-dlx` to the registry or to `@pnpm/registry-mock`.

Reply:
Sure! Let me add to `@pnpm/registry-mock`.

Reply:
I think that's a good idea since it'll make this test faster and should reduce the flake rate.

https://github.com/pnpm/registry-mock/pull/24

Reply:
ok, it will be released in 4.1.0 soon

Reply:
Awesome!